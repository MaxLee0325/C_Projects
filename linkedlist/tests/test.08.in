new
iterator
append alpha
next
append beta
next
append gamma
next
remove
end
