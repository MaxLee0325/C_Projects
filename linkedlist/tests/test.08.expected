[ ]
[ ]
[ alpha ]
Next: alpha
[ alpha ]
[ alpha beta ]
Next: beta
[ alpha beta ]
[ alpha beta gamma ]
Next: gamma
[ alpha beta gamma ]
Removed: 'gamma'
[ alpha beta ]
